components/automate-chef-io/README.md

```diff
@@ -6,25 +6,31 @@ A new static website for Automate is based on [Hugo](https://gohugo.io), the Go-
 
 ### One Time Install
 
-**DISABLE SMART QUOTES**
-OS X tries to use smartquotes by default when editing Hugo docs. You want smartquotes disabled regardless if you are writing code. Here's how [you can disable it](http://www.iclarified.com/38772/how-to-disable-curly-quotes-in-mac-os-x-mavericks).
+1. Disable smart quotes. OS X tries to use smartquotes by default when editing Hugo docs. You want smartquotes disabled regardless if you are writing code. Here's how [you can disable it](http://www.iclarified.com/38772/how-to-disable-curly-quotes-in-mac-os-x-mavericks).
 
-1. Install Hugo
+1. [Install Hugo](https://gohugo.io/getting-started/installing/). If you're using homebrew:
 
     ```
-    $ brew install hugo
+    brew install hugo
     ```
 
-### Routine Setup
+1. [Install go-swagger](https://goswagger.io/install.html). If you're using homebrew:
 
-_These are the steps you'll need to run every time you launch a development environment._
+    ```
```

Comment:
We've installed swagger in the docker container, so we probably don't need these lines.

Reply:
What is the docker container? Is that the preferred way to run hugo? Does this mean we don't recommend installing hugo or go-swagger?

Reply:
Ok, let's go with what you have. We will explain the docker approach in another PR.